homeassistant/components/demo/strings.json

```diff
@@ -63,6 +63,29 @@
     }
   },
   "entity": {
+    "climate": {
+      "demo_climate": {
+        "attribute": {
```

Comment:
Should this be `state_attribute` instead?

Reply:
Maybe we can fully drop it even? Just thinking out loud, for example:

```json
  "entity": {
    "climate": {
      "ubercool": {
        "fan_mode": {
          "name": "Fan mode",
          "state": {
            "auto_high": "Auto High",
            "auto_low": "Auto Low",
            "on_high": "On High",
            "on_low": "On Low"
          }
        },
        "swing_mode": {
          "name": "Swing mode",
          "state": {
            "1": "1",
            "2": "2",
            "3": "3",
            "auto": "Auto",
            "off": "[%key:common::state::off%]"
          }
        },
        "state": {
          "off": "[%key:common::state::off%]",
          "cool": "Ultra Cool",
          "auto": "Cooldown",
          "fan_only": "Some wind"
        }
      }
    }
  }
```

Maybe not the best example ever, but it shows the gist?

Reply:
ok agreed, and `state_attributes` sounds good 👍

Reply:
We are using `state_attributes` in the front end for `preset_mode` translations.